apps/www/pages/privacy-231017.mdx

```diff
@@ -312,10 +312,67 @@ Due to the confidential nature of data processing we may ask you to verify your
 
 We use the following types of cookies:
 
-| Cookie Name     | Type of cookie     | How long does the cookie stay on my device? | Purpose of the cookie |
-| --------------- | ------------------ | ------------------------------------------- | --------------------- |
-| \_ \_stripe_mid | Strictly necessary | 1 year                                      | Fraud Prevention      |
-| \_ \_stripe_sid | Strictly necessary | 30 minutes                                  | Fraud Prevention      |
+<table>
```

Comment:
can we not use markdown?

@pamelachia

Reply:
ok done!

Reply:
ah, sorry, i meant like this

```
| Cookie   | Type of cookie    | When is the cookie dropped?                        | How long does the cookie stay on my device? | Purpose of the cookie                                        | Links to respective privacy notices                                       |
|----------|-------------------|---------------------------------------------------|--------------------------------------------|-------------------------------------------------------------|------------------------------------------------------------------------|
| Stripe   | Strictly necessary | When you access the payment interface on our Service. | 30 minutes                                 | Detecting and preventing fraudulent payments and transactions. | [Stripe Privacy Notice](https://stripe.com/en-gb-sg/cookie-settings)   |
| Stripe   | Strictly necessary | When you access the payment interface on our Service. | 1 year                                     | Detecting and preventing fraudulent payments and transactions. | [Stripe Privacy Notice](https://stripe.com/en-gb-sg/cookie-settings)   |
| Stripe   | Strictly necessary | When you access the payment interface on our Service. | 2 years                                    | Detecting and preventing fraudulent payments and transactions. | [Stripe Privacy Notice](https://stripe.com/en-gb-sg/cookie-settings)   |
| Cloudflare | Strictly necessary | When you first access the Site.                      | 30 minutes                                 | Identify and block bots from accessing the Service.           | [Cloudflare Privacy Notice](https://developers.cloudflare.com/fundamentals/reference/policies-compliances/cloudflare-cookies/) |
| Cloudflare | Strictly necessary | When you first access the Site.                      | End of Session                             | Manage traffic to our Service to ensure that it does not get overloaded. | [Cloudflare Privacy Notice](https://developers.cloudflare.com/fundamentals/reference/policies-compliances/cloudflare-cookies/) |
| Youtube  | Strictly necessary | When you view a Youtube embedded video on our Site.  | End of Session                             | To display embedded videos to you on our Site.                 | [Google Privacy Notice](https://policies.google.com/technologies/cookies) |
| hCaptcha | Strictly necessary | When you respond to a CAPTCHA to prove you are not a robot. | End of Session                             | To record your responses to the CAPTCHA and grant you access to areas of the Service that we do not allow to be accessed by bots. | [hCaptcha Privacy Notice](https://www.hcaptcha.com/privacy)         |

```

Reply:
We actually changed away from this because it looked heavy and repetitive of the same services. And there was no way to merge rows without using <table>

Reply:
yea sure. feel free to do it how you want.